modules/ding_session_cache/ding_session_cache.module

```diff
@@ -86,18 +86,21 @@ function ding_session_cache_set($module, $cid, $data) {
  *   Name of the module that the date is stored under (prefix).
  * @param string $cid
  *   Index data is stored under.
+ * @param bool $default
+ *   Default value if cache for the $cid is empty.
  *
  * @return mixed
  *   If data found it's returned else FALSE.
```

Comment:
This comment should be updated to reflect the added default value.

Reply:
Right you are :-) I have updated the documentation for return as well.